src/main/java/com/java401/AlbumCookies401/controllers/SongController.java

```diff
@@ -0,0 +1,46 @@
+package com.java401.AlbumCookies401.controllers;
+
+import com.java401.AlbumCookies401.models.Album;
+import com.java401.AlbumCookies401.models.Song;
+import com.java401.AlbumCookies401.repostories.AlbumCookiesRepository;
+import com.java401.AlbumCookies401.repostories.SongRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+//@Controller
+//public class SongController {
+//    @GetMapping("/")
+//    public String getSongs(Model model) {
+//        model.addAttribute("songs", ????)
+//        return "splash.html";
+//    }
+//}
+//used demo to template out song controller
+@Controller
+public class SongController {
+
+    @Autowired
+    AlbumCookiesRepository albumCookiesRepository;
+    @Autowired
+    SongRepository songStandRepository;
+
+
+    @PostMapping("/song/add")
+    public RedirectView addSong(String name, Integer songCount, Integer songLength, String trackNumber) {
+        Album album = AlbumCookiesRepository.findByAlbumTitle(albumTitle);
```

Comment:
You'll need to find a way to pass in the albumTitle along with the song information.